### DISCUSSION TITLE
de/1-03-machine-vs-human-learning/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#57] [General]
# de/1-03-machine-vs-human-learning/

https://ai.lernos.org/de/1-03-machine-vs-human-learning/

**Comment by rednaxelaesor** [comment]
Kapitel Modellauswahl: Hier fehlt der Bezug zum Kapitel: "...neuronales Netz (vgl. das **Kapitel....** in diesem Leitfaden)"

**Comment by KathaZ** [comment]
A1/Z1,2 und Z2; A3/Z2: Wauwau
A1/Z11: *Nachfolgend findest du eine s*chematische Darstellung
A2/Z3: I*n* unserem
A3/Z2,3: (vgl. *Kapitel 2* in diesem Leitfaden) => am besten geklammerten Einschub löschen und *neuronales Netz" mit Kapitel verlinken
A5/Z8: Modell *ein* noch tiefergehende*re*s Verständnis
A8/Z7: Benutzer*:innen*,
A9/Z3: Poten*z*iale

**Comment by Myszak64** [comment]
"Modellauswahl" und "Training" ist in der anfänglichen Erklärung fast 100% identisch!!
